content/en/developers/new_tools/add_tool.md

```diff
@@ -155,7 +155,10 @@ If the container doesn't work yet, it's sometimes useful to try and troubleshoot
 16) Edit an entry for your package in your fork of neurocommand/blob/main/neurodesk/apps.json based on one of the other entries (generating one menu item for opening a terminal inside the containers, and one menu item for the GUI, if relevant). Notice that in the json file, the version field should contain the _buildDate_
 17) Include an icon file in your fork of neurocommand/neurodesk/icons
 18) Send a pull request from your fork of neurocommand to https://github.com/NeuroDesk/neurocommand/ 
-19) When the pull request is merged by Neurodesk admins, it will trigger an action to build the singularity container, distribute it in all object storage locations and on CVMFS, and it will update the menus in the desktop image on the next daily build. The new menu entry will be available after 24 hours in neurodesktop:latest and will be announced publicly with the next release.
-20) Check in the dev build if everything is ok before releasing a new version of Neurodesktop:
-<pre class="language-shell command-line" data-prompt="$"><code>sudo docker pull vnmd/neurodesktop-dev:latest && sudo docker run   --shm-size=1gb -it --privileged --name neurodesktop   -v ~/neurodesktop-storage:/neurodesktop-storage   -e HOST_UID="$(id -u)" -e HOST_GID="$(id -g)"   -p 8080:8080 -h neurodesktop-dev   vnmd/neurodesktop-dev:latest</code></pre>
-21) Consider contributing a tutorial about the new tool: https://github.com/NeuroDesk/neurodesk.github.io/tree/hugo-docsy/content/en/tutorials
+19) When the pull request is merged by Neurodesk admins, it will trigger an action to build the singularity container, distribute it in all object storage locations and on CVMFS, and it will update the menus in the desktop image on the next daily build. 
+20) Wait at least 24 hours
+21) Download and run the daily build of neurodesktop to check that your app can be launched from the start menu and works properly:
+<pre class="language-shell command-line" data-prompt="$"><code>sudo docker pull vnmd/neurodesktop:latest && sudo docker run   --shm-size=1gb -it --privileged --name neurodesktop   -v ~/neurodesktop-storage:/neurodesktop-storage   -e HOST_UID="$(id -u)" -e HOST_GID="$(id -g)"   -p 8080:8080 -h neurodesktop-latest   vnmd/neurodesktop:latest</code></pre>
+22) Post an issue in the open an issue in https://github.com/NeuroDesk/neurocontainers/issues notifying that your app appears in the start menu and tested. The app will be included in the next release of Neurodesktop, and will be mentioned in the public announcement that accompanies the release. If the app is not in the start menu or not working as expected based on your earlier testing, open an issue as well, and report it.
+23) If somebody wants to use the application before the next release of Neurodesktop is out, you can instruct them to use the command in step 21 above instead of the deafult commands given in the user install instructions.
```

Comment:
The windows/mac/linux startup binaries will also start the latest version automatically

Reply:
Is that what we want? Don't we want the developer who built the new container to test the final version before people are using it? Let's decide on it, and then the Swinburne team can make the required changes to the binaries (unless we are not familiar with the tools used to develop them).

Reply:
the check of the tool as to happen before it's merged into the apps.json file - after that it's already live on CVMFS

Reply:
I know. But sometimes there are problems in the launch through the menu. Isn't it? (environment variables, folder binding,  etc.) Correct me if wrong.

Reply:
shouldn't - but there is no other way than building the image and testing it

Reply:
I'm not familiar with how neurodesktop:latest is built exactly, but I'll be wary of letting end users use a version that neither of the developers at least tried once. There can always be problems we didn't think of, even in parts other than the menu.

I say to keep the recommendation to use the last release version, and update the launch binaries accordingly.

If you think differently, we can discuss it in our next meeting.

Reply:
I don't mind. I am happy with the launch binaries extracting the latest tested release from our website (but this is currently not implemented) and starting it. Currently the launch binaries start latest, as this was the quickest way to get it working.

Reply:
OK. I noted it down and check if we have the expertise to modify the binaries in Swinburne. If we can do it, I'll update you with the estimated deadline.

Reply:
great! It is straight forward for Linux and Mac, but not for powershell. The latest tested version needs to be extracted from here:
 https://raw.githubusercontent.com/NeuroDesk/neurodesk.github.io/hugo-docsy/data/neurodesktop.toml

Reply:
Hi @stebo85
Just want to update that I was busy with building containers and now will be busy with the talk, so unfortunately, we'll need to put the extraction of last tested version on hold. I don't think that we have an actual issue for that, but in case we do, you're welcome to mark it as "won't fix right now". Sorry for that, but there are indeed higher priorities.